cfgov/regulations3k/tests/test_regdown.py

```diff
@@ -184,6 +184,39 @@ def test_tables_extension_exists(self):
         self.assertIn('<th>First Header</th>', regdown(text))
         self.assertIn('<td>Content Cell</td>', regdown(text))
 
+    def test_no_underscore_emphasis(self):
+        self.assertIn('<em>foo</em>', regdown('*foo*'))
+        self.assertIn('<strong>foo</strong>', regdown('**foo**'))
+        self.assertIn('<strong><em>foo</em></strong>', regdown('***foo***'))
+        self.assertNotIn('<em>foo</em>', regdown('_foo_'))
+        self.assertNotIn('<strong>foo</strong>', regdown('__foo__'))
+        self.assertNotIn('<strong><em>foo</em></strong>', regdown('___foo___'))
+
+    def test_pseudo_form_field_end_of_line(self):
+        text = 'Form field: ___'
+        self.assertIn('Form field: <span class="pseudo-form">___</span>',
+                      regdown(text))
```

Comment:
These EOL form fields are special in that we'll want to swap out the underscores with an underline that fills the width of the page. It'd be helpful if they have a unique class that makes them easy to target via CSS.

One idea: Would it be possible to check for `\n` at the end of the underscores and, if found, append `_extend` to the class (a la `pseudo-form_extend`)?